benchmarks/build.xml

```diff
@@ -80,7 +80,17 @@
 			<path refid="project.classpath" />
 		</path>
 
-		<property name="sample.sql.properties.file" value="benchmarks.properties" />
+		<available file="benchmarks.${user.name}.properties" property="user.properites.avaiable" />
```

Comment:
@slnn 
You include build.properties like this:
`<property file="build.${user.name}.properties" />
	<property file="build.properties" />`
 Why you include benchmark.properties use the same way?

FYI @dantewang 

Tina.

Reply:
Resend in https://github.com/dantewang/liferay-portal/pull/1774

<property file="build.${user.name}.properties" /> <property file="build.properties" />

I set all these import related properties in build.properties file originally, I forgot to remove these unused includes after moving to benchmark.properties.